CONTRIBUTING.adoc

```diff
@@ -612,25 +612,82 @@ vector: '/images/logos/${imageId}/${imageId}.svg'
 credit: 'Your Name'
 credit_url: 'https://twitter.com/yourtwitteraccount'
 ```
+== Maintainer guide
+
+This section contains information for contributors who are interested to help with the Jenkins website maintenance.
+
+=== Website structure and ownership
+
+The Jenkins website hosts various content: user and developer documentation, blog, governance materials, pages for special interest groups and sub-projects, etc.
+This information is maintained by multiple teams.
+Ownership domains are also defined in the link:/.github/CODEOWNERS[CODEOWNERS] file.
+Note that this file might be out of date or missing some entries, so common sense there applies.
+
+Notable special areas:
```

Comment:
Will need to be amended once the hosting team is official to own `/doc/developer/publishing` and `/doc/developer/plugin-governance`.

Reply:
Yes, I will submit it as a follow-up